src/actions.ts

```diff
@@ -368,30 +372,30 @@ export function getKeyDownHandler<Cell: Types.CellBase>(
     handlers = keyDownHandlers;
   }
   return handlers[key];
-}
+};
 
-export function keyDown<Cell: Types.CellBase>(
-  state: Types.StoreState<Cell>,
-  event: SyntheticKeyboardEvent<HTMLElement>
+export function keyDown(
+  state: Types.IStoreState<any>,
+  event: KeyboardEvent<HTMLInputElement>
 ) {
-  const handler = getKeyDownHandler<Cell>(state, event);
+  const handler = getKeyDownHandler(state, event);
   if (handler) {
     return handler(state, event);
   }
   return null;
 }
 
-export function dragStart<T>(state: Types.StoreState<T>) {
+export function dragStart() {
```

Comment:
- [ ] Bring types back

Reply:
The function doesn't really use the parameter, so I thought of deleting it. but yea, might be better to just leave it.